components/planner/src/dynamo/planner/utils/prometheus.py

```diff
@@ -35,15 +35,15 @@ def _get_average_metric(
         increase(metric_sum[interval])/increase(metric_count[interval])
 
         Args:
-            metric_name: Base metric name (e.g., 'nv_llm_http_service_inter_token_latency_seconds')
+            metric_name: Base metric name (e.g., 'dynamo_frontend_inter_token_latency_seconds')
```

Comment:
If we're passing in the metric name without the `dynamo_frontend_` prefix to this function, then this example string should be `inter_token_latency_seconds`